MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -12,6 +12,8 @@
                                disregard_event = cms.bool(False),
                                use_tracks = cms.bool(True),
                                tracks_src = cms.InputTag('jmtRescaledTracks'),
+                               match_jets = cms.bool(False),     
+                               match_jet_src = cms.InputTag('selectedPatJets'),  
```

Comment:
Are these used anywhere, or can they be dropped?

Reply:
it was there from Ang's study of allowing seed tracks from matched jets. They are usually unused as he did no seem to fine it work better than our sigm_dxy >= 4 in his study?

Reply:
It doesn't show up anywhere else in the code (not even in the C++ anywhere), so I think this part of the config can be dropped then. Just wanted to confirm with you before doing so!

I'll remove it now.